packages/dd-trace/src/noop/proxy.js

```diff
@@ -2,14 +2,17 @@
 
 const NoopTracer = require('./tracer')
 const NoopAppsecSdk = require('../appsec/sdk/noop')
+const { NoopDogStatsDClient } = require('../dogstatsd')
```

Comment:
Let's move it to a separate file so that the real client is not even imported when not needed.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/4225/commits/8ac4d6cdfe6be90fcf9119c6b49b68ef49fed835